ansible/setup-nodes.yml

```diff
@@ -4,4 +4,9 @@
   become: yes
 
   roles:
-     - role: lablabs.rke2
\ No newline at end of file
+     - role: upgrade_machine
+     - role: lablabs.rke2
+       vars: 
+         rke2_download_kubeconf: true
+         rke2_download_kubeconf_file_name: rke2.yaml
+         rke2_download_kubeconf_path: /etc/rancher/rke2
```

Comment:
This needs to be `/tmp` unfortunately or similar, as we need to run it with `sudo` to write to /etc and we should not run Ansible as root